src/contact/containers/EditInfo.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import EditInfo from '../components/EditInfo';
+import * as actions from '../actions/Actions';
+import { bindActionCreators } from 'redux';
+
+
+const mapStateToProps = (state) => {
+	return {
+		info : state.reducer.details,
+		hasError: true,
+		error: null,
+		isEditable: state.reducer.isEditable
+	}
+};
+
+const mapDispatchToProps = (dispatch) => {
+	return {
+		actions : bindActionCreators(actions, dispatch)
+	}
+};
+
+export default connect(mapStateToProps, mapDispatchToProps)(EditInfo);
```

Comment:
Make use of propTypes, which I see missing.

Reply:
Yes, earlier, I had this in my mind, but it is skipped. I will add.